glibc.yaml

```diff
@@ -494,22 +467,21 @@ subpackages:
           mkdir -p "${{targets.subpkgdir}}"/sbin
           mv "${{targets.destdir}}"/sbin/sln "${{targets.subpkgdir}}"/sbin
 
-  - name: "libcrypt1"
-    description: "Password hashing library included with glibc"
-    dependencies:
-      provider-priority: 10
-    pipeline:
-      - runs: |
-          mkdir -p "${{targets.subpkgdir}}"/lib
-          mv "${{targets.destdir}}"/lib/libcrypt.so* "${{targets.subpkgdir}}"/lib/
-
   - name: "glibc-locale-posix"
     description: "POSIX locale data for glibc"
     pipeline:
       - runs: |
           mkdir -p "${{targets.subpkgdir}}"/usr/lib/locale
           mv "${{targets.destdir}}"/usr/lib/locale/C.utf8 "${{targets.subpkgdir}}"/usr/lib/locale/
 
+  - name: "libcrypt1"
+    description: "Password hashing library included with glibc"
```

Comment:
This package should no longer exist and be dropped. Everything that used to depend on libcrypt1 should transition to libxcrypt.

Reply:
I can't remember how we did the split transition / migration in other distros.